### PR TITLE
adding black formatting

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

since the project already sets `isort` formatting with black it make sense to also use black code formating

## Before submitting

- [ ] Did you read the [contributing guideline](https://github.com/omni-us/jsonargparse/blob/master/CONTRIBUTING.rst)?
- [ ] Did you update **the documentation**? (readme and public docstrings)
- [ ] Did you write **unit tests** such that there is 100% coverage on related code? (required for bug fixes and new features)
- [x] Did you verify that new and existing **tests pass locally**?
- [x] Did you make sure that all changes preserve **backward compatibility**?
- [ ] Did you update **the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

### CODE REVIEW

**Comment by Borda** [comment]
probably would be better first land #280

File: jsonargparse/namespace.py [hunk 1]
```diff
@@ -44,9 +44,13 @@ def is_meta_key(key: str) -> bool:
 
 
 @overload
-def strip_meta(cfg: 'Namespace') -> 'Namespace': ...
+def strip_meta(cfg: "Namespace") -> "Namespace":
+    ...
```
Comment (mauvilsa) [inline jsonargparse/namespace.py:48]:
```suggestion
    ...  # pragma: no cover
```

File: jsonargparse/namespace.py [hunk 2]
```diff
@@ -44,9 +44,13 @@ def is_meta_key(key: str) -> bool:
...
+
+
 @overload
-def strip_meta(cfg: Dict[str, Any]) -> Dict[str, Any]: ...
+def strip_meta(cfg: Dict[str, Any]) -> Dict[str, Any]:
+    ...
```
Comment (mauvilsa) [inline jsonargparse/namespace.py:53]:
```suggestion
    ...  # pragma: no cover
```

**Review by mauvilsa — APPROVED** [review]
Thank you!

**Pull request merged**